modules/python/init.zsh

```diff
@@ -46,6 +46,20 @@ if (( $? && $+commands[virtualenvwrapper.sh] )); then
   source "$commands[virtualenvwrapper.sh]"
 fi
 
+# Load PIP completion.
+if (( $+commands[pip] )); then
+  cache_file="${0:h}/cache.zsh"
+
+  if [[ "$commands[pip]" -nt "$cache_file" || ! -s "$cache_file" ]]; then
+    # pip is slow; cache its output. And also support 'pip2', 'pip3' variants
+    pip completion --zsh | sed -e "s|compctl -K _pip_completion pip|& pip2 pip3|" >! "$cache_file" 2> /dev/null
```

Comment:
I assume this may cause oddities for pip2 vs pip3 if the user has both installed... because the pip command can only point to one of them... I'm not sure what the correct method of handling this would be though.